app/domain/SlpFileWriter.js

```diff
@@ -69,6 +69,13 @@ export default class SlpFileWriter extends EventEmitter {
     };
   }
 
+  getFullBuffer() {
+    if (this.currentFile.buffersToConcat.length > 0) 
+      this.currentFile.fullBuffer = Buffer.concat([this.currentFile.fullBuffer, ...this.currentFile.buffersToConcat]);
```

Comment:
at this point the question is how fast is this line? I might need to run some tests to get a baseline.

Reply:
Well i have measurements now and they don't look good for the old method.

https://twitter.com/xMetonym/status/1285995190881148929

Reply:
Given a Buffer of about 20MB concating with a small 8 byte buffer it takes 8ms, at 40MB it takes 11ms, at 64MB it takes 17ms. This means the old method would continually get worse as you passed the 40ish minute mark. This new method adds a small stutter (a few frames) if the user connects in the middle of the game, but doesn't affect the game otherwise.